docs/quickstart.md

```diff
@@ -45,7 +45,7 @@ id: helloworld
 states:
 - id: hello
   type: noop
-  transform: '{ msg: ("Hello, " + .name + "!") }'
+  transform: 'jq({ msg: ("Hello, " + .name + "!") })'
```

Comment:
Not 100% sure, because I'm in a different branch and cant test right now but think you do this:
```yaml
transform: 
  msg: "Hello jq(.name) !"
```

Should do the same thing, but shows the benefit of new jq better and string interpolation